plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -0,0 +1,2589 @@
+                - self.validated_config: If successful, a validated version of the 'config' parameter.
+        Example:
+            To use this method, create an instance of the class and call 'validate_input' on it.
+            If the validation succeeds, 'self.status' will be 'success' and 'self.validated_config'
+            will contain the validated configuration. If it fails, 'self.status' will be 'failed', and
+            'self.msg' will describe the validation issues.
+        """
+
+        temp_spec = {
+            'syslog_destination': {
+                'type': 'dict',
+                'name': {'type': 'str'},
+                'description': {'type': 'str'},
+                'host': {'type': 'str'},
+                'protocol': {'type': 'str'},
+                'port': {'type': 'int'},
+            },
+            'snmp_destination': {
+                'type': 'dict',
+                'name': {'type': 'str'},
+                'description': {'type': 'str'},
+                'ip_address': {'type': 'str'},
+                'port': {'type': 'str'},
+                'snmp_version': {'type': 'str'},
+                'community': {'type': 'str'},
+                'username': {'type': 'str'},
+                'snmp_mode': {'type': 'str'},
+                'snmp_auth_type': {'type': 'str'},
+                'auth_password': {'type': 'str'},
+                'snmp_privacy_type': {'type': 'str'},
+                'privacy_password': {'type': 'str'},
+            },
+            'email_destination': {
+                'type': 'dict',
+                'primary_smtp_config': {
+                    'type': 'dict',
+                    'hostname': {'type': 'str'},
+                    'port': {'type': 'str'},
+                    'smtp_type': {'type': 'str'},
+                    'username': {'type': 'str'},
+                    'password': {'type': 'str'},
+                },
+                'secondary_smtp_config': {
+                    'type': 'dict',
+                    'hostname': {'type': 'str'},
+                    'port': {'type': 'str'},
+                    'smtp_type': {'type': 'str'},
+                    'username': {'type': 'str'},
+                    'password': {'type': 'str'},
+                },
+                'from_email': {'type': 'str'},
+                'to_email': {'type': 'str'},
+                'subject': {'type': 'str'},
+            },
+            'rest_webhook_destination': {
+                'type': 'dict',
+                'name': {'type': 'str'},
+                'description': {'type': 'str'},
+                'url': {'type': 'str'},
+                'method': {'type': 'str'},
+                'trust_cert': {'type': 'bool'},
+                'headers': {
+                    'type': 'list',
+                    'elements': 'dict',
+                    'name': {'type': 'str'},
+                    'value': {'type': 'str'},
+                    'default_value': {'type': 'str'},
```

Comment:
Value assigned to the HTTP header.
Default value for the HTTP header that can be used if no specific value is provided.
Since API is supporting default value seperately but in GUI it's not like that.